packages/runner/src/suite.ts

```diff
@@ -275,7 +275,7 @@ function createTest(fn: (
 }
 
 function formatName(name: string | Function) {
-  return typeof name === 'string' ? name : name instanceof Function ? name.name : String(name)
+  return typeof name === 'string' ? name : name instanceof Function ? (name.name || 'Anonymous') : String(name)
```

Comment:
Shouldn't it be `<anonymous>`? This is what we do for serialized error:

https://github.com/vitest-dev/vitest/blob/158c4bb056417828833088508b43e6917d65d23f/packages/utils/src/error.ts#L29

Reply:
Ok.